src/test/resources/load_config_test.properties

```diff
@@ -0,0 +1,23 @@
+# You can enter as many threads to stimulate a load test. A single user is represented by each Thread.
+# So if you wish to simulate a load test with 5 concurrent users then you need to enter 5 as the value
+# for this property.
+number.of.threads=3
+
+# It indicates the time taken to create all of the threads needed to fore the requests.
+# If you set 10 seconds as the ramp-up period for 5 threads then the framework will take 10 seconds to create those 5 threads,
+# i.e. each thread will be in use appx 2 secs gap between the requests.
+# Also by setting its value to 0 all the threads can be created at once at the same time.
+# Note- If you want to ramp up and fire more threads/user-requests in less time e.g. 5 threads in 2secs(or 1 sec),
+# then, use '@UseHttpClient(SslTrustHttpClient.class)' as this closes the connection before making next connection.
+ramp.up.period.in.seconds=3
+
+# Keep it equal or lesser than "number.of.threads" size
+#During the ramp up period, say 10 sec, how many requests you want to fire.
+#e.g. in 10 secs, I want to fire 5 requests, that means, if you loop 6 times ie loop.count=6
+#you fire 5*(60/10)=30 per min i.e. 30*60 = 1800 per hour i.e. 1800*8=14400 per 8hrs
+requests.during.ramp.up.period=5
+
+# By specifying its value framework gets to know that how many times the test(s) will be repeated
+# e.g.
+loop.count=2
+
```

Comment:
please introduce a property e.g.
`continuous.run.time.in.seconds`
This will tell the framework for how many minutess or hours(1hr=60min=3600seconds) to run.
`continuous.run.time.in.seconds=3600`

Reply:
#72 has been raised for this.